server/src-lib/Hasura/RQL/Types/ScheduledTrigger.hs

```diff
@@ -0,0 +1,103 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Hasura.RQL.Types.ScheduledTrigger
+  ( ScheduleType(..)
+  , CreateScheduledTrigger(..)
+  , RetryConfST(..)
+  , formatTime'
+  ) where
+
+import           Data.Time.Clock
+import           Data.Time.Clock.Units
+import           Data.Time.Format
+import           Data.Aeson
+import           Data.Aeson.Casing
+import           Data.Aeson.TH
+import           Hasura.Prelude
+import           System.Cron.Types
+import           System.Cron.Parser
+import           Hasura.Incremental
+
+import qualified Data.Text                     as T
+import qualified Data.Aeson                    as J
+import qualified Hasura.RQL.Types.EventTrigger as ET
+
+data RetryConfST
+  = RetryConfST
+  { rcstNumRetries  :: !Int
+  , rcstIntervalSec :: !Seconds
+  , rcstTimeoutSec  :: !Seconds
+  , rcstTolerance   :: !NominalDiffTime
+  } deriving (Show, Eq, Generic)
+
+instance NFData RetryConfST
+instance Cacheable RetryConfST
+
+$(deriveJSON (aesonDrop 2 snakeCase){omitNothingFields=True} ''RetryConfST)
+
+defaultRetryConf :: RetryConfST
+defaultRetryConf =
+  RetryConfST
+  { rcstNumRetries = 1
+  , rcstIntervalSec = seconds 10
+  , rcstTimeoutSec = seconds 60
```

Comment:
This is quite a nitpick, but it seems odd to write `seconds 60` instead of `minute 1`. :)

Reply:
HTTP timeouts are usually used in seconds, no?

Reply:
Are they? That seems like an implementation detail: timeouts are durations of time. It doesn’t matter what units they’re described in.

Reply:
Haha, I dunno. But most of our APIs take seconds as the input (hence the generalization). Hence felt natural to use seconds for a field called `rcstTimeoutSec`. 

That's my only argument but if you prefer `minute 1`, I don't think I have any issues. But I wanted to make this argument before folding.

Reply:
Yes, all things being equal, I would name the field `rcstTimeout`, not `rcstTimeoutSec`! 😄 But I realize you’re using `rcstTimeoutSec` and `Seconds` here because you’re deriving the `FromJSON`/`ToJSON` instances, which is reasonable.

As for whether to write `seconds 60` or `minutes 1`, I really don’t care much one way or the other. If you think `seconds 60` is clearer, leave it as `seconds 60`!

Reply:
Yay :)